Sources/Fuzzilli/Modules/Statistics.swift

```diff
@@ -75,9 +78,23 @@ public class Statistics: Module {
             self.ownData.totalExecs += 1
             self.currentExecs += 1
         }
-        fuzzer.registerEventListener(for: fuzzer.events.InterestingProgramFound) { _ in
+        fuzzer.registerEventListener(for: fuzzer.events.InterestingProgramFound) { ev in
             self.ownData.interestingSamples += 1
             self.ownData.coverage = fuzzer.evaluator.currentScore
+
+            if !ev.program.runtimeTypes.isEmpty {
```

Comment:
I think it would be a little nicer if this was a dedicated getter like `hasRuntimeTypeInformation`. Also I'm wondering if checking for `isEmpty` is always correct. Couldn't there be cases where runtime type collection had succeeded, but didn't collect any types, e.g. because the program is empty or is all dead code or something like that? Maybe there should be an explicit enum member in `Program` that captures the state of runtime type information, which could probably be either `.notCollected`, `.failedToCollect`, or `.available`. Then you could also automatically determine whether to collect type information in `processInteresting`. WDYT?

Reply:
Maybe you could even reuse the `TypeCollectionResult` enum for this?

Reply:
Actually, if you do that then you don't even need to change the `InterestingProgramFound` event, so that would also be kind of cool!

Reply:
yeah I thought there should not be many interesting empty programs (and was too lazy? 🤷‍♂️ )
but with `typeCollectionResult` it should be easy to check this
I guess we have to add `typeCollectionResult` to protobuf too.

Reply:
> I guess we have to add `typeCollectionResult` to protobuf too.

Yes, let's do that! :)

Maybe it should be called `typeCollectionStatus` since it can be `.notAttempted` (the default)? Both are fine with me though.